content/docs/1.4.0/volumes-and-nodes/storage-tags.md

```diff
@@ -28,10 +28,18 @@ When creating a volume, specify the disk tag and node tag in the UI.
 
 ### Kubernetes
 
-Use Kubernetes StorageClass setting to specify tags.
+Use Kubernetes StorageClass setting to specify tags to schedule replicas only on the matching nodes.
```

Comment:
```suggestion
Use Kubernetes StorageClass setting to specify tags.
```
I removed `to schedule replicas only on the matching nodes.`, because this section is not just for node but also disk.

Reply:
Yes, I think it can be achieved if setting the label like `disk=ssd`.  So I will follow the comment to remove the words.